src/descriptors/power_system_structs.json

```diff
@@ -3020,118 +3044,136 @@
       "supertype": "AVR"
     },
     {
-      "struct_name": "AVRTypeII",
-      "docstring": "Parameters of an Automatic Voltage Regulator Type II -  Typical static exciter model",
+      "struct_name": "ESDC2A",
+      "docstring": "Is used to represent field-controlled dc commutator exciters with continuously acting voltage regulators having power supplies derived from the generator or auxiliaries bus.. \nParameters of IEEE Std 421.5 Type DC2A Excitacion System. This model corresponds to ESDC2A in PSSE and PSLF",
```

Comment:
```suggestion
      "docstring": "Is used to represent field-controlled dc commutator exciters with continuously acting voltage regulators having power supplies derived from the generator or auxiliaries bus.\nParameters of IEEE Std 421.5 Type DC2A Excitacion System. This model corresponds to ESDC2A in PSSE and PSLF",
```

Reply:
Something to consider: we could detect long lines in the code generation script and add line breaks.  This might add consistency.  You could still add them in places where you want itemized lists.

Reply:
I think we tried that and the issue was it breaks JSON syntax

Reply:
I don't follow.  The JSON strings would not have new line characters.  The script would detect long lines and add new line characters in the generated Julia comments.